server/cmd/server/.env

```diff
@@ -12,3 +12,9 @@ MULTIVERSX_PROXY="https://testnet-gateway.multiversx.com"
 BRIDGE_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
 # Max retrials to wait for account nonce update after sending bridge txs
 MAX_RETRIALS_WAIT_NONCE=60
+# Server certificate for tls secured connection with clients.
```

Comment:
also here why not a config.toml file?

Reply:
Several reasons:
- for small projects or simple setups, using a single .env file can be straightforward and easy to manage.
- docker Compatibility: .env files are commonly used in Docker containers, and they integrate well with container orchestration tools like Docker Compose and Kubernetes. They allow for easy configuration changes without modifying the application code, which is very useful.

There is already a task created to make it runnable with either .env/configs/flags.